### PR TITLE
Added level up/down to Items By Points Earned

### PR DESCRIPTION
**Pull request by bgsavage**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.ts [hunk 1]
```diff
@@ -201,6 +201,10 @@ export class AssessmentResultsComponent implements OnInit {
     return GradeCode.getIndex(gradeCode);
   }
 
+  getPointRowStyleClass(index: number){
+    return index == 0 ? 'level-down' : '';
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.ts]: conditionally add the class 'level-down' only on the first points column.

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html [hunk 2]
```diff
@@ -213,7 +213,7 @@ <h3 class="gray-darkest mb-md">
               <span [hidden]="!showValuesAsPercent">{{ item.fullCreditAsPercent | number:'1.0-0' }}%</span>
             </ng-template>
           </p-column>
-          <p-column *ngFor="let column of pointColumns" [field]="column.numberField" header="{{'labels.groups.results.assessment.items.cols.x-points' | translate:{id: column.points} }}" [sortable]="true">
+          <p-column *ngFor="let column of pointColumns; let i = index" [field]="column.numberField" header="{{'labels.groups.results.assessment.items.cols.x-points' | translate:{id: column.points} }}" [sortable]="true" [styleClass]="getPointRowStyleClass(i)">
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: I'm not sure how we can add the 'row-end' class here, as we need access to the row data at the p-column level, which I don't think we have.  I talked to @aaronsleeper and he said he might have a css solution on all empty `<td>`s here.

Comment (jtreuting) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: could it be done in the body template instead or is that what you are saying as the work around

Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: The body template is in isde the `<td>` in which @aaronsleeper said he wouldn't be able to traverse up tot he <tr> from there.  But I think Aaron has a css fix for this.

**Pull request merged**